### DISCUSSION TITLE
好書分享_被科技綁架的智慧城市

### DISCUSSION THREAD

**Discussion by MAKERRRofKEVIN** [1101-GS4719-C/Python-Community-GS4719-C#179] [General]
去年讀的好書，作者針對了近幾年的一些新興科技方案提出一些隱藏的問題，並提出相當有深度的見解與解方，其中最令我印象深刻的要屬作者指出近幾年愈發興盛的預測性警務系統背後隱藏的警方執法偏見，因為這類人工智慧系統相當仰賴數據的支撐，但警方執法的對象難免受到「認為哪些人更容易犯罪」影響，故這類系統做出的指示往往是加強平民窟、有色人種區的執法，但常此以往只是更加深執法的偏差與偏見。

對此作者指出了一個相當有深度的實例，有些執法單位開始轉而將數據提供給社福單位，透入資源到預測犯罪的區域而非投入警力，確實能夠達到降低犯罪率及執法的效果，這類對科技的反思與創新應用確實相當值得我等參考學習。

博客來購書連結：https://www.books.com.tw/products/0010867296

**Comment by Jie2222** [comment]
感謝同學分享，不論是沉浸在新的科技當中，或希望以科技改變未來，我們都需要透徹的釐清其價值和意義，否則套用一句前人說的話:這是最好的時代，也是最壞的時代。

**Comment by HSU-YU-JEN** [comment]
這本書的觀點很棒~像是換個角度，利用這些數據從根本改善犯罪成因，而非利用它形成更多對立與偏見

**Comment by Tomlord1122** [comment]
數據是一回事，重要的還是人的思維成長要跟上科技的快速進步

**Comment by ghost** [comment]
我很認同這本書提到的觀點!雖然數據一定會包含地緣的影響，但過於重視地緣的影響反而容易造成模型失真；書中提到從根源解決的辦法是非常值得嘗試的辦法

**Comment by Isongzhe** [comment]
感謝同學分享，我覺得這本書所提及的 非常值得探討。
